benchmarks/kernels/benchmark_moe_align_block_size.py

```diff
@@ -87,8 +32,8 @@ def check_correctness(num_tokens, num_experts=256, block_size=256, topk=8):
         x_names=["num_tokens", "num_experts", "topk"],
         x_vals=configs,
         line_arg="provider",
-        line_vals=["vllm", "triton"],  # "triton"
-        line_names=["VLLM", "Triton"],  # "Triton"
+        line_vals=["vllm"],
+        line_names=["vLLM"],
```

Comment:
Should we move the kernel into the benchmark script? It seems not that useful to have a benchmark as just one kernel

Reply:
I think moving the kernel into the benchmark doesn't help because we already win the triton kernel for every shape, so we don't need to maintain it. In the future when new kernel comes, just compare with the current version would be good,  what's your thought?

Reply:
Okay, that is fair enough. As long as we have a naive implementation to unit test against, it should be okay

Reply:
Yes, someone else developed a torch version for unit test. But it is quite slow, so I don't choose to compare it for benchmark.